openshift-knative-operator/cmd/operator/kodata/knative-serving/0.19.0/2-serving-core.yaml

```diff
@@ -2264,6 +2264,13 @@ spec:
           volumeMounts:
             - mountPath: /etc/tls/private
               name: secret-activator-sm-service-tls
+          resources:
+            requests:
+              memory: 20Mi
+              cpu: 10m
+            limits:
+              memory: 40Mi
```

Comment:
Are these limits necessary and non-dangerous? Containers will be killed I think if they reach this.

Reply:
I know but I just followed what are the recommendations in the link above given what rbac does it should use minimal mem/cpu. I could remove them if we think it can cause issues.

Reply:
FWIW we don't set any limits for kube-rbac-proxy in cluster monitoring operator, only requests. We had the general recommendation to avoid setting memory limits for our components but Serg has more context.

Reply:
If just requests makes the HPA happy, let's do that and not take the extra risk.

Reply:
Ok good to know I will remove them then.